SDF-to-Coords

```diff
@@ -0,0 +1,49 @@
+import os
+
+import numpy as np
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
+                xyz = []
+                counter = 0
+                for line in f:
+                    counter += 1
+                    if counter < 4:
+                        continue
+                      
+                    elif counter > 4 and counter <= 4+num_atom:
+                        xyz.append(line.split()[0:4])
+                        continue
+                        
+                    elif counter > 4+num_atom:
+                        pass
+                    
+            xyz = np.asarray(xyz)
+            for atom in xyz:
+                i = [3,0,1,2]
+                xyz = xyz[:,i]
+                break
+            return(xyz)
+
+#This is different code; gives input for funct_grp using xyz
+
+xyz = SDF_to_coords(filename, num_atoms)
+
+def funct_grp_coords(fnct_atoms):
+    funct_grp = []
+    for i in fnct_atoms: 
+        for atom in xyz:
+            i = int(i)
+            i = xyz[i,]
+            funct_grp.append(i)
+            break
+    funct_grp = np.asarray(funct_grp)
+    
+    return funct_grp
+
+funct_atoms = []
+
+funct_grp_coords(funct_atoms)
```

Comment:
No problems with this code either. Although it's a tad wonky, because the input for the function is the index number of the row that the desired atomic symbol is in.

Reply:
I think referring to atoms by the input order is the best you're going to do. As a piece of advice I strongly recommend you pass 'xyz' into your function and not rely on it being defined elsewhere. Especially with Python, it's super easy to reuse an identifier (set 'xyz' to something else) and unintentionally change your functions behavior.